src/components/Task/Task.jsx

```diff
@@ -0,0 +1,41 @@
+const React = require('react')
```

Comment:
Instead of adding this new class/file, you could overwrite TaskListItem that got added in https://github.com/cheshire137/gh-notifications-snoozer/pull/16.